pablobascunana/youml-api
# ML-44 Implement JWT

## User story
[Jira ML-44](https://youml.atlassian.net/browse/ML-44)

## Changes

- Implement JWT to verify user
- Add active user function to verify user
- Change verification URL
